File: src/main/java/com/example/glamourstudio/entities/Atendimento.java
package com.example.glamourstudio.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Atendimento {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String cliente;

    private String profissional;

    private LocalDateTime dataHora;

    private String servico;

    private String situacaoAtendimento;

    private EnumStatusAtendimento statusAtendimento = EnumStatusAtendimento.ATIVO;


}

File: front/app/types/atendimento.ts
export class Atendimento{

    constructor(

        public id:number | null,

        public cliente:string,

        public profissional:string,

        public dataHora:string,

        public servico:string,

        public statusAtendimento:string

    ){}

}

export interface AtendimentoFormProps{

    atendimentoExistente?:Atendimento

}